algorithms/Java/sorting/bubble-sort.java

```diff
@@ -1,27 +1,41 @@
+package JAVA;
```

Comment:
Welcome @shivam-0510 

Thanks for the good work

Please remove this line `package JAVA;` as it is only required for your local project.

and please add some information about the new optimized algorithm in a multi-line comment.